pkg/sql/logictest/testdata/logic_test/sqllite

```diff
@@ -449,3 +449,12 @@ SELECT pk, col0 FROM tab64784 WHERE (col0 IN (SELECT col3 FROM tab64784 WHERE co
 4  216
 1  213
 0  212
+
+# Regression test for crashing when intertwining GetRequests and
```

Comment:
Can we create another execbuilder test that shows the requests generated, so that if the commands generated are updated we'll notice?